iXBRLViewerPlugin/viewer/src/js/calculation.js

```diff
@@ -18,13 +18,16 @@ export class Calculation {
         const fact = this.fact;
         const report = fact.report;
         if (!this._conceptToFact) {
-            const rels = report.getChildRelationships(fact.conceptName(), "calc")
             const ctf = {};
-            for (const [elr, rr] of Object.entries(rels)) {
-                ctf[elr] = {};
-                if (rr.length > 0) {
-                    const otherFacts = report.getAlignedFacts(fact, {"c": rr.map(r => r.t )});
-                    otherFacts.forEach(otherFact => setDefault(ctf[elr], otherFact.conceptName(), new FactSet()).add(otherFact));
+            for (const version of ["calc", "calc11"]) {
```

Comment:
Should it be `["calc"]` only if `this.calc11` is false?

Reply:
I was wondering what to do about this as I didn't want the calculations to simply disappear if you disable Calc 1.1 mode.

I've just pushed 65af4b2 which adds an explicit notice in this case:

![image](https://github.com/Arelle/ixbrl-viewer/assets/45077928/d18e1047-b6ec-466f-b232-bd90980f9369)